srv/components/controller/files/scripts/provision.sh

```diff
@@ -1747,3 +1747,73 @@ do_provision()
             echo "No provisioning details found on the controller"
     }
 }
+
+ntp_status_get()
+{
+    ## objects name in the xml
+    _xml_obj_bt="ntp-status"
+    # Property value to be extracted from xml
+    _xml_obj_plist=("ntp-status" "ntp-server-address" "ntp-contact-time")
```

Comment:
Why not read JSON output?

Reply:
the utility already has implementation for the xml parseer